.github/workflows/ci.yml

```diff
@@ -10,12 +12,13 @@ jobs:
       matrix:
         node-version: [12.x]
```

Comment:
Just a FYI 14 is the stable LTS version of node. Please use that for development